numba_dppy/tests/test_dpnp_functions.py

```diff
@@ -267,6 +264,732 @@ def f(a):
             self.assertTrue(np.array_equal(expected, got))
 
 
+@unittest.skipUnless(ensure_dpnp(), "test only when dpNP is available")
+class Testdpnp_random_functions(unittest.TestCase):
+    sizes = [None, 9, (2, 5), (3, 2, 4)]
+
+    def test_random_sample(self):
+        from numba.tests.support import captured_stdout
+
+        @njit
+        def f(size):
+            c = np.random.random_sample(size)
+            return c
+
+        sizes = [9, (2, 5), (3, 2, 4)]
+        set_dpnp_debug(1)
+        with captured_stdout() as got_gpu_message:
+            with dpctl.device_context("opencl:gpu"):
+                for size in sizes:
+                    result = f(size)
+                    _result = result.ravel()
+                    for i in range(_result.size):
+                        self.assertTrue(_result[i] >= 0.0)
+                        self.assertTrue(_result[i] < 1.0)
+                    self.assertIn("DPNP implementation", got_gpu_message.getvalue())
+        set_dpnp_debug(None)
+
+    def test_ranf(self):
+        from numba.tests.support import captured_stdout
```

Comment:
Should be imported at the top of the file once.

Reply:
Correct.